CMakeLists.txt

```diff
@@ -616,6 +616,26 @@ if (WITH_OPENH264 AND NOT OPENH264_FOUND)
 endif()
 set(WITH_OPENH264 ${OPENH264_FOUND})
 
+option(WITH_MS_RDPECAM_CLIENT "Build with video camera redirect client" OFF)
+if(WITH_MS_RDPECAM_CLIENT)
+	# currently camera redirect client supported for platforms with Video4Linux only
+	find_package(V4L)
```

Comment:
`find_package(V4L REQUIRED)` already does the failure if not found

Reply:
Fixed